apps/website/src/components/Playground/index.tsx

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
 import { SandpackProvider, SandpackLayout, SandpackCodeEditor, SandpackPreview } from '@codesandbox/sandpack-react';
 import { useColorMode } from '@docusaurus/theme-common';
-import AppCode from '!!raw-loader!./template/App.js';
-import StylesCode from '!!raw-loader!./template/Styles.js';
+import { useLocation } from '@docusaurus/router';
+import AppCode from '!!raw-loader!./code/app.js';
+import DefaultStylesCode from '!!raw-loader!./code/styles.js';
+import path from 'path-browserify';
+
+const ctx = require.context('!!raw-loader!./code/templates', false, /\.js$/);
+
+const templates: Record<string, string> = ctx.keys().reduce((acc, modulePath) => {
+  if (modulePath.includes('app')) {
+    return acc;
+  }
+
+  const templateName = path.parse(modulePath).name;
```

Comment:
AFAIR Webpack normalizes paths, so we can replace this module with `templateName.split('/').slice(-1)[0]`

Reply:
had to strip the extension as well, but yeah that works